shoop/admin/generated_resources.txt

```diff
@@ -0,0 +1,9 @@
+static/shoop_admin/js/picotable.js
```

Comment:
It is not autogenerated. It's a file that tells which resources will be generated by "npm run build" so that build_resources.py knows what to cache.